### PR TITLE
Approved - Add reset button

### PR DESCRIPTION
**Pull request by TommiJuuti**
Added reset button to reset counter

### CODE REVIEW

File: app/src/main/java/com/example/testdummy/helloworld/MainActivity.java [hunk 1]
```diff
@@ -60,4 +60,12 @@ public void countMore(View view) {
         String message = "Click count: " + count;
         counterText.setText(message);
     }
+
+    public void resetCounter(View view) {
+        count = 0;
+
+        EditText counterText = (EditText) findViewById(R.id.textCounter);
+        String message = "Click count: " + count;
```
Comment (zsombi) [inline app/src/main/java/com/example/testdummy/helloworld/MainActivity.java]: Shouldn't this be a localized string?

**Comment by zsombi** [comment]
Please see the comments inline. Otherwise the code looks cool. Great contribution, thanks!

File: app/src/main/java/com/example/testdummy/helloworld/MainActivity.java (hunk 1 above)
Comment (TommiJuuti) [inline app/src/main/java/com/example/testdummy/helloworld/MainActivity.java]: Usually yes, but this is just a test app with no plans to localize, thus static text used.

Comment (zsombi) [inline app/src/main/java/com/example/testdummy/helloworld/MainActivity.java]: Ack. Thanks, I learned something about Android today ;)

**Pull request merged**